### DISCUSSION TITLE
Add option to only partially remove windows defender...

### DISCUSSION THREAD

**Discussion by DavidXanatos** [ionuttbara/windows-defender-remover#158] [Ideas]
... remove it to the extent you can actually uninstall it on windows server through "features and roles"
this means remove only the antivirus component but leave the UI for firewall exploit protection HVCI and alike.
Ann those are nice optional features only the Antivirus is malicious and interferes with the users wishes.

**Comment by BeegDeek** [comment]
Windows Firewall is still there. Any competent antivirus should block any firewall exploit. I use Malwarebytes and it's amazing. I haven't got infected since 2008 after I installed it. Windows Defender is garbage
